demo/webpack.config.js

```diff
@@ -1,321 +1,339 @@
-const { join, relative, resolve, sep } = require("path");
+const { join, relative, resolve, sep } = require('path');
+const fs = require('fs');
 
-const webpack = require("webpack");
-const nsWebpack = require("nativescript-dev-webpack");
-const nativescriptTarget = require("nativescript-dev-webpack/nativescript-target");
-const { getNoEmitOnErrorFromTSConfig } = require("nativescript-dev-webpack/utils/tsconfig-utils");
-const CleanWebpackPlugin = require("clean-webpack-plugin");
-const CopyWebpackPlugin = require("copy-webpack-plugin");
+const webpack = require('webpack');
+const nsWebpack = require('@nativescript/webpack');
+const nativescriptTarget = require('@nativescript/webpack/nativescript-target');
+const { getNoEmitOnErrorFromTSConfig } = require('@nativescript/webpack/utils/tsconfig-utils');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
-const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer");
-const { NativeScriptWorkerPlugin } = require("nativescript-worker-loader/NativeScriptWorkerPlugin");
-const TerserPlugin = require("terser-webpack-plugin");
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const { NativeScriptWorkerPlugin } = require('nativescript-worker-loader/NativeScriptWorkerPlugin');
+const TerserPlugin = require('terser-webpack-plugin');
 const hashSalt = Date.now().toString();
 
-module.exports = env => {
-    // Add your custom Activities, Services and other Android app components here.
-    const appComponents = env.appComponents || [];
-    appComponents.push(...[
-        "tns-core-modules/ui/frame",
-        "tns-core-modules/ui/frame/activity",
-        resolve(__dirname, "app/activity.android.ts")
-    ]);
+module.exports = (env) => {
+	const platform = env && ((env.android && 'android') || (env.ios && 'ios') || env.platform);
+	if (!platform) {
+		throw new Error('You need to provide a target platform!');
+	}
 
-    const platform = env && (env.android && "android" || env.ios && "ios" || env.platform);
-    if (!platform) {
-        throw new Error("You need to provide a target platform!");
-    }
+	const platforms = ['ios', 'android'];
+	const projectRoot = __dirname;
 
-    const platforms = ["ios", "android"];
-    const projectRoot = __dirname;
+	if (env.platform) {
+		platforms.push(env.platform);
+	}
 
-    if (env.platform) {
-        platforms.push(env.platform);
-    }
+	// Default destination inside platforms/<platform>/...
+	const dist = resolve(projectRoot, nsWebpack.getAppPath(platform, projectRoot));
 
-    // Default destination inside platforms/<platform>/...
-    const dist = resolve(projectRoot, nsWebpack.getAppPath(platform, projectRoot));
+	const {
+		// The 'appPath' and 'appResourcesPath' values are fetched from
+		// the nsconfig.json configuration file.
+		appPath = 'src',
```

Comment:
Should it not be 
appPath = 'app',
appResourcePath = 'app/App_Resources',

in your case?

Reply:
The appPath and appResourcePath should be picked up from nativescript.config.ts from NS7. Aldo, I will chage it here just to not have confusion.